nixos/modules/misc/ids.nix

```diff
@@ -275,6 +275,8 @@
       gocd-server = 252;
       terraria = 253;
       mattermost = 254;
+      quagga = 255;
```

Comment:
Thank you for clarifying the ID allocation policy, it makes sense to try to preserve IDs for services with stateful data.

Quagga does not store any stateful data, it uses the 'quagga' user&group just to drop privileges after startup and to control access to its unix sockets in /run/quagga; no other files should have owner 'quagga'. 
So in this specific case the dynamic user IDs are appropriate.